Gemfile

```diff
@@ -6,5 +6,7 @@ source "https://rubygems.org"
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
+gem 'attr_encrypted', github: "PagerTree/attr_encrypted", branch: "rails-7-0-support"
```

Comment:
we should open a ticket to come back after a while and re-visit this. Going to a random branch is not very reliable

Reply:
I can attempt to replicate the change on our own fork if that is preferable. Now that I think about it... I realise we have no control over whether someone decides to re-write the git history of this branch and suddenly the hash in our Gemfile.lock does not exist... not ideal.